app/src/main/java/com/example/sharingang/DetailedItemFragment.kt

```diff
@@ -78,6 +85,13 @@ class DetailedItemFragment : Fragment() {
         return binding.root
     }
 
+    private fun initBuy() {
```

Comment:
Could you please add the documentation?

Reply:
This one is private, so it is not required to document, but can be done if you're not sure what the function does.

Reply:
I thought we were also supposed to document private methods.

Reply:
I don't know, but maybe, this is what they put in their code review "All public classes, methods and fields must be documented."

Reply:
Hmm indeed.